ssl/ssl_ciph.c

```diff
@@ -1971,3 +1971,61 @@ int SSL_CIPHER_get_digest_nid(const SSL_CIPHER *c)
         return -1;
     return ssl_cipher_table_mac[i].nid;
 }
+
+int SSL_CIPHER_get_key_exchange(const SSL_CIPHER *c)
+{
+    if (c == NULL)
```

Comment:
In FOO_bar APIs, we don't check if FOO is NULL. It's the application's responsibility.